stages/org.osbuild.timezone

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/python3
+
+import json
+import subprocess
+import sys
+
+
+def main(_tree, options):
+    zone = options["zone"]
+
+    subprocess.run(["rm", "/etc/localtime"], check=True)
```

Comment:
This will fail if `/etc/localtime` does not exist. Do we want this, so that we find out when we can remove the line?

Either way, we want a comment explaining why we chose the way we did.

Reply:
please take a look on the explanation